applications/TopologyOptimizationApplication/tests/Small_Cantilever/ProjectParameters.json

```diff
@@ -0,0 +1,123 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "Small_Cantilever_Hex",
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 1.0
+    },
+    "solver_settings"          : {
+        "solver_type"                        : "topology_optimization_simp_static_solver",
+        "model_part_name"                    : "Structure",
+        "domain_size"                        : 3,
+        "echo_level"                         : 0,
+        "analysis_type"                      : "linear",
+  	"problem_domain_sub_model_part_list" : ["Parts_structure"],
+        "processes_sub_model_part_list"      : ["DISPLACEMENT_disp","PointLoad3D_load"],
+        "model_import_settings"              : {
+            "input_type"     : "mdpa",
+            "input_filename" : "Small_Cantilever_Hex"
+        },
+        "material_import_settings"           : {
+            "materials_filename" : "StructuralMaterials.json"
+        },
+        "time_stepping"                      : {
+            "time_step" : 1.1
+        },
+        "line_search"                        : false,
+        "convergence_criterion"              : "residual_criterion",
+        "displacement_relative_tolerance"    : 0.0001,
+        "displacement_absolute_tolerance"    : 1e-9,
+        "residual_relative_tolerance"        : 0.0001,
+        "residual_absolute_tolerance"        : 1e-9,
+        "max_iteration"                      : 10,
+        "rotation_dofs"                      : false,
+        "compute_reactions"                  : true,
+        "reform_dofs_at_each_step"           : false,
+        "move_mesh_flag"                     : true,
+        "linear_solver_settings"             :  {
+            "solver_type"         : "amgcl",
```

Comment:
you sure abt this one?
We have very good direct solvers now in the LinearSolversApp (e.g. pardiso)

Reply:
For the small example pardiso will prob.l be faster, but for larger volume meshes, the iterative solver is faster.

Reply:
I am not sure, but if I use "pardiso" as solver I get an error. The error states:
```
RuntimeError: Error: Trying to construct a Linear solver with solver_type:
"pardiso" which does not exist.
The list of available options (for currently loaded applications) is:
Kratos components
    amgcl
    amgcl_ns
    bicgstab
    cg
    deflated_cg
    monotonicity_preserving
    scaling
    skyline_lu_factorization
    sparse_cg
    sparse_lu
    sparse_qr
    tfqmr


in kratos/factories/linear_solver_factory.h:108:typename LinearSolver<TSparseSpace, TLocalSpace>::Pointer LinearSolverFactory<TSparseSpace, TLocalSpace>::Create(Parameters) const [with TSparseSpace = UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::ublas::vector<double> >; TLocalSpace = UblasSpace<double, boost::numeric::ublas::matrix<double>, boost::numeric::ublas::vector<double> >; typename LinearSolver<TSparseSpace, TLocalSpace>::Pointer = shared_ptr<LinearSolver<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::ublas::vector<double> >, UblasSpace<double, boost::numeric::ublas::matrix<double>, boost::numeric::ublas::vector<double> > > >]
```

Reply:
seems like you didn't enable MKL:
https://github.com/KratosMultiphysics/Kratos/tree/master/applications/LinearSolversApplication#enable-mkl-optional

in the meantime you could also use the `sparse_lu`

FYI selecting the fastest available direct linear solver: 
https://github.com/KratosMultiphysics/Kratos/blob/7e6dafcfef17b0ef351cd5c8589adf520767e3d3/kratos/python_scripts/python_linear_solver_factory.py#L34-L43